src/store/initial-state.js

```diff
@@ -1,25 +1,73 @@
-import deepmerge from 'deepmerge';
-import { loadState, saveState } from './helpers';
+import { mergeLocalStorage } from './helpers';
 import normalizeData from './normalize-data';
 import { getFlagsFromUrl, Flags } from '../utils/flags';
 import { sidebarWidth } from '../config';
 
-/**
- * Create new default state instance for properties that aren't overridden
- * when the pipeline is reset with new data via the App component's data prop
- * @return {object} state
- */
-export const createInitialState = () => ({
+export const initialState = {
```

Comment:
Great to see all of the state in one place here. Optional but I think I'd prefer to keep this wrapped in a factory function though for protection rather than risk a single mutable reference.